modules/manage/pages/security/encryption.adoc

```diff
@@ -10,7 +10,15 @@ By default, Redpanda data is sent unencrypted. A security best practice is to en
 * Transport Layer Security (TLS), previously SSL, provides encryption for client-server communication. A server certificate prevents third parties from accessing data transferred between the client and server.
 * mTLS, or 2-way TLS, is a protocol that authenticates both the server and the client. In addition to the server certificate required in TLS, mTLS also requires the client to give a certificate. This involves more overhead to implement, but it can be useful for environments that require additional security and only have a small number of verified clients.
 
-For each Redpanda broker, specify the key file (`broker.key`), the certificate file (`broker.crt`), and the truststore file (`ca.crt`). Each broker has its own `broker.key` and `broker.crt`, but all brokers can have the same `ca.crt`.
+For each Redpanda broker, you must specify the:
+
+* key file (`broker.key`)
+* certificate file (`broker.crt`)
+* truststore file (`ca.crt`). 
+
+You can also specify a certificate revocation list (`ca.crl`) to block connections from entities using certifications that have expired or been revoked by a certificate authority (CA). 
```

Comment:
```suggestion
You can also specify a certificate revocation list (`ca.crl`) to reject connections from entities using certifications that have been revoked by a certificate authority (CA). 
```

Reply:
@michael-redpanda, I've updated the intro so that it's clearer there is only TLS support for CRLs.